Mbarak-Mbigo/cp2_bucketlist
# Ft sphinx documentation

#### What does this PR do?
Add sphinx documentation
#### Description of Task to be completed?
Add sphinx documentation
#### How should this be manually tested?
run the following command: `docs/$ ./manage.py make build`
Then access the index page on docs directory with a browser
Expected results: Sphinx generated documentation
#### Any background context you want to provide?
Integrating feedback from Shem
